package.json

```diff
@@ -10,6 +10,7 @@
     "url": "https://github.com/bbatliner/vex-dialog"
   },
   "dependencies": {
+    "deep-clone-simple": "^1.1.1",
```

Comment:
Let's use the exact version, `1.1.1`, for more consistent dependency management.